src/GlobalAccountHeader/GlobalAccountHeader.react.js

```diff
@@ -0,0 +1,171 @@
+import classnames from 'classnames';
+import Component from 'react-pure-render/component';
+import Favorite from './Favorite.react';
+import LinkBase from '../LinkBase/LinkBase.react';
+import LoginLink from '../LoginLink/LoginLink.react';
+import React, { PropTypes } from 'react';
+import ToggleNav from '../ToggleNav/ToggleNav.react';
+
+const messages = {
+  favorites: 'My Favorites',
+  logIn: 'Log In',
+  register: 'REGISTER',
+  support: 'Support',
+  freeTrialCta: 'START FREE TRIAL',
+  renewCta: 'RENEW',
+  upgradeCta: 'UPGRADE'
+};
+
+export default class GlobalHeaderAccount extends Component {
+  static propTypes = {
+    deviceType: PropTypes.string,
+    includeAccount: PropTypes.bool,
+    optionalComponent: PropTypes.func,
+    user: PropTypes.object.isRequired
+  }
+
+  static defaultProps = {
+    includeAccount: false
+  }
+
+  renderUser() {
+    const { deviceType, includeAccount, user } = this.props;
+    const favorites = messages.favorites;
+    const classNames = 'global-header-account__link button tomato';
```

Comment:
let's see if we can strip the `tomato` references from here - Maybe a default style for this button in the associated style sheet that references the hex variable name. Putting the `tomato` here was my doing - ties the presentation to the implementation (bad Jason!).